### ISSUE TITLE
"Transfer" and "claim rewards" buttons are not accessible on mobile device

### ISSUE THREAD

**Issue by espoem** [busyorg/busy#1059]
If a user uses a mobile device for browsing Busy, he is not able to transfer currency nor claim rewards in the wallet page or in the user's profile. If they go to another user profile, there is an option to send the funds to the user.

![image.png](https://res.cloudinary.com/hpiynhbhq/image/upload/v1510241834/sjrek43jzadggwbeksiy.png)

![image.png](https://res.cloudinary.com/hpiynhbhq/image/upload/v1510242432/ehzivzfktqdrgrlkq8ps.png)

![image.png](https://res.cloudinary.com/hpiynhbhq/image/upload/v1510242125/lzkkr1rlrg9a1takdnsv.png)

### Expected behaviour

It should be possible to transfer or claim rewards in the wallet page on mobile devices. This regards the users profile too.

![image.png](https://res.cloudinary.com/hpiynhbhq/image/upload/v1510241798/hbb47hnzreg8ta3fcvvd.png)

### Steps to reproduce

1. Go to https://busy.org/wallet on mobile

- Browser: Brave
- OS: Android 7.1.2

/n/n<br /><hr/><em>Open Source Contribution posted via <a href="https://utopian.io/utopian-io/@espoem/transfer-and-claim-rewards-buttons-are-not-accessible-on-mobile-device">Utopian.io</a></em><hr/>

**Comment by jm90m** [comment]
@pareshUD What should we do about this?

**Comment by pareshUD** [comment]
@jm90m As I mentioned to Fabien also we need to have transfer and wallet button throughout the website.. We need made this in newer design for current version we can make wallet page more accessible by changing the placement of the wallet icon in top menu... What do you think?

**Comment by jm90m** [comment]
yeah we can make the wallet icon in top menu, can you create an issue with design and details and assign it to me @pareshUD ?

**Comment by pareshUD** [comment]
@jm90m  here is the design for wallet design for mobile.. ALso check the top bar for mobile where we can have the icon on top..

![busy homepage mobile](https://user-images.githubusercontent.com/8911074/34662730-b69c1456-f478-11e7-867c-f912d5de9ec7.png)
![busy wallet mobile](https://user-images.githubusercontent.com/8911074/34662731-b6d60c2e-f478-11e7-8a5c-031d6a8764a1.png)

**Comment by jm90m** [comment]
@pareshUD looks good, is transfer button always the footer? and its fixed?

**Comment by bonustrack** [comment]
What about cla8m reward button and market price?

**Comment by pareshUD** [comment]
![busy wallet mobile](https://user-images.githubusercontent.com/8911074/34979261-d3d0146a-fac6-11e7-893e-26b585a34893.png)

@bonustrack check this and let me know
